visualforce-page1/question 4/LookUpOnOpportunityController.apxc

```diff
@@ -0,0 +1,37 @@
+public class LookUpOnOpportunityController {
+
+    public Opportunity opportunity{get;set;}
+     public Id contactId{get;set;}
```

Comment:
Indentation is not proper in the classes.

Reply:
Done